Authentication/AuthenticationBootstrapper.swift

```diff
@@ -219,33 +235,29 @@ public class AuthenticationBootstrapper<User: UserType, SessionService: SessionS
 extension AuthenticationBootstrapper: LoginControllerTransitionDelegate {
     
     public func onSignup(controller: LoginController) {
```

Comment:
When the user is in the login screen and presses the `Sign Up` button. The documentation branch adds this explanation here.